### DISCUSSION TITLE
[com_fields] Missing "Searchable" column in #__fields database table

### DISCUSSION THREAD

**Discussion by machadoug** [joomla/joomla-cms#14872] [2. Cold Features]
Adding a "searchable" field in com_fields will allow us to better integrate with the search plugins and with other extensions. In my case I only want some fields to be "searchable" therefore adding a "Searchable" column in #__fields database table allow us to change the search plugin and add `->where('f.searchable = 1')` in /plugins/search/content/content.php after line 189.

Is this something that can be done?

**Comment by laoneo** [comment]
Is not a bad idea.

**Comment by SemaphoreOxalis** [comment]
Not sure how this can replace a filter/ordering option in menu item...

**Comment by Hackwar** [comment]
This has been implemented in Joomla 5.0 for Smart Search.
